### PR TITLE
Testimonials

### PR DESCRIPTION
**Pull request by chayok17**
nav buttons upd

### CODE REVIEW

File: sections/feedback.php [hunk 1]
```diff
@@ -23,11 +23,12 @@
...
             </div>
             <button class="carousel-control-prev" type="button" data-bs-target="#carouselTestimonials" data-bs-slide="prev">
-                <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                <span class="carousel-control-prev-icon" style="background-color: black;" aria-hidden="true"></span>
                 <span class="visually-hidden">Previous</span>
             </button>
+
             <button class="carousel-control-next" type="button" data-bs-target="#carouselTestimonials" data-bs-slide="next">
-                <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                <span class="carousel-control-next-icon" style="background-color: black;" aria-hidden="true"></span>
```
Comment (puricemaxim) [inline sections/feedback.php:31]: please, use class not style
class="bg-dark" for dark background.

**Review by puricemaxim — CHANGES_REQUESTED** [review]
please, remove style and use class, see comments ( bg-dark )

File: sections/STA.php [hunk 2]
```diff
@@ -2,6 +2,8 @@
     <div class="container text-center">
         <h2 class="mb-4" style="font-size: 2.5rem; font-weight: bold;">Хотите узнать больше?</h2>
         <p class="mb-4" style="font-size: 1.2rem;">Запишитесь на бесплатную консультацию с нашими экспертами</p>
-        <a href="#contact-form" class="btn btn-lg btn-outline-dark" style="padding: 15px 30px; border-radius: 50px;">Записаться сейчас</a>
+        <button class="btn btn-lg btn-outline-dark" style="padding: 15px 30px; border-radius: 50px;" data-bs-toggle="modal" data-bs-target="#askQuestionModal">
```
Comment (puricemaxim) [inline sections/STA.php:5]: please, remove style and use bootstrap class

for padding:
https://getbootstrap.com/docs/5.0/utilities/spacing/#margin-and-padding

for border-radius:
https://getbootstrap.com/docs/5.0/utilities/borders/#sizes

**Review by puricemaxim — CHANGES_REQUESTED** [review]
remove style and use class

File: sections/STA.php (hunk 2 above)
Comment (chayok17) [reply sections/STA.php:5]: done

**Pull request merged**